addon/modifiers/sortable-item.js

```diff
@@ -743,12 +740,11 @@ export default class SortableItemModifier extends Modifier {
     // Instead of using `event.preventDefault()` in the 'primeDrag' event,
```

Comment:
So the idea is that if a handle isn't specified, the entire element itself is draggable by default. To disable dragging in general would be done via `isDraggingDisabled`. Just want to make sure that behavior is retained.

Reply:
This PR contains no such changes. It only fixes the handle in the modifier version, which has been broken compared to the non-modifier version. This is a mere bug fix and should not introduce any change in behaviour. The logic is identical to the non modifier version now. Looks like this was work in progress previously, as the comment I removed indicated: 

> // todo what does this do? why do you have to null handle